Rewrite send response BBEs (#3769)
ballerina-platform/ballerina-distribution: examples/http-send-different-status-codes/http_send_different_status_codes.md

# REST service - Send different status codes

> The subtypes of the `http:StatusCodeResponse` record represent different HTTP status code responses. Returning them from the resource function results in the relevant HTTP status code response. To send a non-entity body response, use the relevant constant value declared in the `http` module. These constant values can be directly returned from the resource method as long as the relevant return type is specified in the resource function signature. Use these when you want to send different status code responses without a body and headers.

Reviewer: ```suggestion
The subtypes of the `http:StatusCodeResponse` record type represent different HTTP status code responses. Returning them from the resource function results in the relevant HTTP status code response. To send a non-entity body response, use the relevant constant value declared in the `http` module. These constant values can be directly returned from the resource method by specifying the relevant return type in the resource function signature. Use this when different status code responses need to be sent without a body and headers.
```

Author: Fixed via [9c3c207](https://github.com/ballerina-platform/ballerina-distribution/pull/3769/commits/9c3c207d6ca8971110fd8bce55edd27c8b13e18b)
